vcd/datasource_vcd_catalog.go

```diff
@@ -62,6 +62,36 @@ func datasourceVcdCatalog() *schema.Resource {
 				Computed:    true,
 				Description: "Key and value pairs for catalog metadata",
 			},
+			"catalog_version": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "Catalog version number.",
+			},
+			/*"owner_name": { // owner_name is not available because catalog user view doesn't have this value
```

Comment:
Just checking for my understanding - is it just AdminCatalog which has it?

Reply:
Following @vbauzysvmware recommendation, after retrieving CatalogRecord, I can grab the owner info from there. Now is set, please check :)